CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * [ENHANCEMENT] Query-scheduler: add `cortex_query_scheduler_enqueue_duration_seconds` metric that records the time taken to enqueue or reject a query request. #5879
 * [ENHANCEMENT] Query-frontend: add `cortex_query_frontend_enqueue_duration_seconds` metric that records the time taken to enqueue or reject a query request when not using the query-scheduler. #5879
 * [ENHANCEMENT] Expose `/sync/mutex/wait/total:seconds` Go runtime metric as `go_sync_mutex_wait_total_seconds_total` from all components. #5879
+* [ENHANCEMENT] Nginx: listen on IPv6 addresses. #5948
```

Comment:
Sorry, should go into the helm chart changelog under operations/helm/charts/mimir-distributed

Reply:
We have some documentation on this as well: https://github.com/grafana/mimir/blob/main/docs/internal/contributing/contributing-to-helm-chart.md  , hope it's useful